README.md

```diff
@@ -1,9 +1,9 @@
 Picolarium
 ==========
 
-Picolarium is a [PICO-8](https://www.lexaloffle.com/pico-8.php) clone of the game _Polarium_ for DS.
+Picolarium is a [PICO-8](https://www.lexaloffle.com/pico-8.php) clone of the game _Polarium_ for DS and GBA.
 
-Draw a single stroke, flip tiles from black to white or vice versa, and turn each single line into a solid color each.
+Draw a single stroke, flip tiles from black to white or vice versa, and make horizontal lines of the same colour.
```

Comment:
I'm a little unsure of how to best communicate the fact that each horizontal line (row) needs to be one solid color, but that the lines don't all need to be the _same_ color. That's what my original wording was meant to convey (but I'm not sure I succeeded). Do you feel your change makes this clearer than it was?